great_international/blocks/investment_atlas.py

```diff
@@ -0,0 +1,106 @@
+"""Streamfield blocks for Investment Atlas pages"""
+
+from django import forms
+
+from wagtail.core import blocks
+from wagtail.images.blocks import ImageChooserBlock
+
+from core.widgets import MarkdownTextarea
+from core.helpers import render_markdown
+
+
+DEFAULT_IMAGE_RENDITION_SPEC = "fill-960x540"
+
+
+class BaseOpportunityImageBlock(blocks.StructBlock):
+    image = ImageChooserBlock(required=True)
+    image_alt = blocks.CharBlock(
+        max_length=255,
+        required=True,
+    )
+    caption = blocks.CharBlock(
+        max_length=255,
+        required=False,
+    )
+
+    def get_api_representation(self, value, context=None):
+        """Expand the default serialization to show the image rendition URL"""
+        if not context:
+            context = {}
+
+        retval_dict = {}
+
+        img = value.get("image")
+
+        if img:
+            _rendition_spec = context.get('rendition_spec', DEFAULT_IMAGE_RENDITION_SPEC)
+            retval_dict.update(
+                {
+                    'image': img.get_rendition(_rendition_spec).url,
+                    'image_alt': value.get('image_alt', ''),
```

Comment:
`image_alt` is required and also .get will give null string if empty
```suggestion
                    'image_alt': value.get('image_alt'),
```